File: src/com/bulletphysics/collision/shapes/ConeShapeX.java
package com.bulletphysics.collision.shapes;

/**
 * ConeShape implements a cone shape, around the X axis.
 * 
 * @author jezek2
 */
public class ConeShapeX extends ConeShape {

	public ConeShapeX(double radius, double height) {
		super(radius, height);
		setConeUpIndex(0);
	}

}

File: src/com/bulletphysics/collision/shapes/ConeShapeX.kt
package com.bulletphysics.collision.shapes

/**
 * ConeShape implements a cone shape, around the X axis.
 *
 * @author jezek2
 */
class ConeShapeX(radius: Double, height: Double) : ConeShape(radius, height) {
    init {
        coneUpIndex = 0
    }
}
